exporter/exporterhelper/queued_retry.go

```diff
@@ -152,10 +152,17 @@ func (rs *retrySender) send(req request) (int, error) {
 		return rs.nextSender.send(req)
 	}
 
-	expBackoff := backoff.NewExponentialBackOff()
-	expBackoff.InitialInterval = rs.cfg.InitialInterval
-	expBackoff.MaxInterval = rs.cfg.MaxInterval
-	expBackoff.MaxElapsedTime = rs.cfg.MaxElapsedTime
+	// Cannot use NewExponentialBackOff because if update the InitialInterval
+	// need to reset again, and an unnecessary clock Now is called.
```

Comment:
```suggestion
	// Do not use NewExponentialBackOff since it calls Reset and the code here must
	// call Reset after changing the InitialInterval (this saves an unnecessary call to Now).
```